### ISSUE TITLE
Can't update to new version on Mac

### ISSUE THREAD

**Issue by chrissshe1** [a85/POSTMan-Chrome-Extension#10197]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the Issue

My Postman shows a red dot indicating it has updates
![image](https://user-images.githubusercontent.com/66974056/128404070-f46ddd79-b64d-4b5d-8b26-9a3bec607086.png)

If I click it, Postman closes immediately. I then manually restart the app. It's still on the same version. After a while, the red dot would appear again.

I'm aware of the access issue and have moved Postman from `/Applications` to `~/Applications`

### Steps To Reproduce

1. Click the "Update downloaded, restart now" button following the red dot indicator
2. Restart the app (manually. I don't know if this can be automated but bugged)
3. It's not updated

### Environment Information

```markdown
- Operating System: MacOS big sur 11.5.1 (20G80)
- Platform Type: native app
- Postman Version: 8.8.0
```

**Comment by postman-staff** [comment]
Hi there,

Thanks for reaching out and sorry to see you are having trouble updating your App.

To troubleshoot further we will need App logs, your email ID & other details. Since GitHub being a public platform it is not good to get those details on this thread from you for security reasons.

Could you open a new ticket on our support channel using this link - https://www.postman.com/support/, please? Also, do forget to add this GitHub issue number and your GitHub username in the request description to make it easy for us to track.

Meanwhile, to get you unblocked please feel free to use Postman on Web using any of the below links. Postman on Web has the same UI & feature sets as that of the desktop/native app and let me know if that works for you.

- http://go.postman.co/
- http://web.postman.co/

We will wait to hear back from you on our support channel.

via @Swapnil-getpostman | Postman Support

**Comment by postman-staff** [comment]
Hi there,

My apologies, I forgot to add in my previous response that - you can also download & install the latest version of Postman's desktop from the downloads page using this link - https://www.postman.com/downloads/.

Please make sure to save/sync all your work before you download & install the latest version from the download page link. Do let me know if you face any issues.

via @Swapnil-getpostman | Postman Support

**Comment by chrissshe1** [comment]
Thank you @postman-support , I submitted a support ticket. If I download and install the latest version, will I be able to keep the `current value` in my `Environment`? I think current values are designed to be not synced or exported.

**Comment by postman-staff** [comment]
Hi Chris,

I'm going to go ahead close this issue since you have opened a support ticket.

Have a great day! :)

via @nrmartell | Postman Support
